swagger.yaml

```diff
@@ -28,33 +29,31 @@ paths:
         schema:
           $ref: "#/definitions/Trail"
```

Comment:
Your `GET` /trails endpoint response looks good since its in this format:
```
{
  "data": [
    {
      "id": 0,
      "type": "string",
      "attributes": {
        "id": 0,
        "name": "Dan's Trail",
        "zipCode": 0,
        "difficulty": "Green",
        "attributes": {
          "largeDrop": true,
          "smallDrop": true,
          "woodRide": true,
          "skinny": true,
          "largeJump": true,
          "smallJump": true,
          "gap": true
        }
      }
    }
  ]
}
```
You have two attributes objects though, one nested inside of the other. I would just put all those nested attributes with the other fields in the top level attributes if you want to.

However a similar json structure should be used for endpoints that accept or return a single trail object. For example, the response body of `GET` /trails/ID or `POST` /trails should be something like:
```
{
  "data":  {
      "id": 0,
      "type": "string",
      "attributes": {
        "id": 0,
        "name": "Dan's Trail",
        "zipCode": 0,
        "difficulty": "Green",
        "attributes": {
          "largeDrop": true,
          "smallDrop": true,
          "woodRide": true,
          "skinny": true,
          "largeJump": true,
          "smallJump": true,
          "gap": true
        }
      }
  }
}
```